packages/types/lib/driver.ts

```diff
@@ -662,7 +661,7 @@ export interface Core<C extends Constraints = BaseDriverCapConstraints> {
  * `Ctx` would be the type of the element context (e.g., string, dictionary of some sort, etc.)
  */
 export interface Driver<
-  C extends Constraints = BaseDriverCapConstraints,
+  C extends Constraints = Constraints,
```

Comment:
this is basically "I don't care", which is fine.  we fold the _base_ caps into driver-specific caps in the `DriverCaps`, `NSDriverCaps` and `W3CDriverCaps` types, which 3p drivers are expected to use.